f2/src/forms/ContactInfoForm.js

```diff
@@ -1,162 +1,160 @@
-              <Flex direction="row" align="center" wrap="wrap" mb={10}>
-                <P w="100%">
-                  <Trans id="contactinfoPage.skipInfo" />
-                </P>
-                <Button
-                  as={ReactRouterLink}
-                  fontSize={{ base: 'lg', md: 'xl' }}
-                  color="black"
-                  variant="solid"
-                  variantColor="gray"
-                  bg="gray.400"
-                  borderColor="gray.500"
-                  to="/confirmation"
-                  textAlign="center"
-                >
-                  <Trans id="locationinfoPage.skipButton" />
-                  <Icon
-                    focusable="false"
-                    ml={2}
-                    mr={-2}
-                    name="chevron-right"
-                    size="28px"
+      >
+        {({ handleSubmit, handleChange, handleBlur, errors, submitCount }) => (
+          <Form onSubmit={handleSubmit}>
+            <Container>
+              <Row className="form-section">
+                {Object.keys(errors).length > 0 && (
+                  <ErrorSummary
+                    errors={createErrorSummary(errors)}
+                    submissions={submitCount}
+                    title={<Trans id="contactinfoPage.hasValidationErrors" />}
                   />
-                </Button>
-              </Flex>
-            ) : null}
-            <Field
-              name="fullName"
-              label={<Trans id="contactinfoPage.fullName" />}
-              errorMessage={<Trans id="contactinfoForm.fullName.warning" />}
-              component={Input}
-              required={!fyiForm}
-            />
+                )}
+
+                {errors && errors.enterContactDetails && (
+                  <P color="#dc3545" fontSize="1.25rem" marginBottom="0.5rem">
+                    {enterContactDetails.errorMessage}
+                  </P>
+                )}
+                {errors && errors.fullName && (
+                  <P color="#dc3545" fontSize="1.25rem" marginBottom="0.5rem">
+                    {fullName.errorMessage}
+                  </P>
+                )}
+                {errors && errors.email && (
+                  <P color="#dc3545" fontSize="1.25rem" marginBottom="0.5rem">
+                    {email.errorMessage}
+                  </P>
+                )}
+                {errors && errors.phone && (
+                  <P color="#dc3545" fontSize="1.25rem" marginBottom="0.5rem">
+                    {phone.errorMessage}
+                  </P>
+                )}
+
+                <br />
+                <br />
+
+                {fyiForm ? (
+                  <Flex direction="row" align="center" wrap="wrap" mb={10}>
+                    <P w="100%">
+                      <Trans id="contactinfoPage.skipInfo" />
+                    </P>
+                    <SkipButton
+                      label={<Trans id="locationinfoPage.skipButton" />}
```

Comment:
This is not how the SkipButton component is intended to work. Please review f2\src\components\formik\button\index.js (line 91 - 108). The only props to pass to it are **to** (the page to skip to) and **label** (the text for the button).

Reply:
Resolved in new commit